pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -291,7 +293,10 @@ public boolean execute()
     File[] files = dir.listFiles(new FilenameFilter() {
       @Override
       public boolean accept(File dir, String name) {
-        return name.toLowerCase().endsWith(_format.toString().toLowerCase());
+        if (_format != null) {
```

Comment:
do we want to validate the record reader class, eg: try to instantiate it and make sure it's valid?

Reply:
Didn't get notified about your comments on this PR :( - sorry about the delay in response.

Let me check what the exception path message is to the users - if its sensible enough, I will just let it take that path instead of doing an additional validation.